static/app/components/replays/unmaskAlert.tsx

```diff
@@ -1,40 +1,52 @@
 import styled from '@emotion/styled';
 
 import {Button} from 'sentry/components/button';
+import ExternalLink from 'sentry/components/links/externalLink';
+import useUserViewedReplays from 'sentry/components/replays/useUserViewedReplays';
 import {IconClose, IconInfo} from 'sentry/icons';
-import {t} from 'sentry/locale';
+import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import useDismissAlert from 'sentry/utils/useDismissAlert';
 
-const LOCAL_STORAGE_KEY = 'replay-player-dom-alert-dismissed';
+const LOCAL_STORAGE_KEY = 'replay-unmask-alert-dismissed';
 
-function PlayerDOMAlert() {
+function unmaskAlert() {
   const {dismiss, isDismissed} = useDismissAlert({key: LOCAL_STORAGE_KEY});
+  const {data, isError, isPending} = useUserViewedReplays();
 
-  if (isDismissed) {
+  if (isDismissed || isError || isPending || (data && data.data.length > 3)) {
```

Comment:
so they'll view 3 replays within the timeframe, after that we'll hide the banner. but when the timeframe restarts they'll see it again...

they can dismiss it forever though (via localstorage).

Reply:
i didn't specify a timeframe in the query, so the timeframe is whatever the default is in the endpoint. but that brings up a good point - is it weird for the banner to come back after a period of time?